code/modules/reagents/reagents/drugs.dm

```diff
@@ -316,7 +316,8 @@
 	reagent_state = LIQUID
 	color = "#181818"
 	overdose = REAGENTS_OVERDOSE
-	addiction_chance = 0 //Anything above 0 will have 100% odds when smoking
+	addiction_chance = 1 //Anything above 0 will have 100% odds when smoking
```

Comment:
Is this intented? we gone back and forth like 8 times now on this

Reply:
Yes, you and I haven't afaik? The idea here is to give some ups and downs to nicotine since it's the most common means of sanity gain for a lotta people.

Reply:
Alrighto